### PR TITLE
Issue #92 update last login

### PR DESCRIPTION
**Pull request by Askaholic**
Not sure where to set the last login. This works, but I don't think it's ideal. It should probably use the time of the request, but I don't know where that code is. 

Addresses #92

### CODE REVIEW

**Comment by bukajsytlos** [comment]
Isn't this responsibility of API (https://github.com/FAForever/faf-java-api
)?

**Comment by Brutus5000** [comment]
@bukajsytlos Currently not. In the current protocol there is still no usage of a oauth JWT for login. Thus you could login to the lobby without logging in to the API.

File: faf-java-server-app/src/main/java/com/faforever/server/security/Login.java [hunk 1]
```diff
@@ -52,6 +55,10 @@
   @Column(name = "ip")
   private String ip;
 
+  @Temporal(TemporalType.TIMESTAMP)
+  @Column(name = "last_login")
+  private Date lastLogin;
```
Comment (Brutus5000) [inline faf-java-server-app/src/main/java/com/faforever/server/security/Login.java]: `java.util.Date` is not used in this project as it was superceeded by the `java.time` package. Please use `java.time.OffsetDateTime` (the `@Temporal` is not required on java.time types)
See `AbstractEntity` as a reference.

**Review by Brutus5000 — CHANGES_REQUESTED** [review]
We should also add a unit test.

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**